lib/decision_review_v1/utilities/form_4142_processor.rb

```diff
@@ -13,14 +13,15 @@ class Form4142Processor
       # @return [Hash] the generated request body
       attr_reader :request_body
 
-      def initialize(form_data:)
+      def initialize(form_data:, submission_id: nil)
         @form = form_data
         @pdf_path = generate_stamp_pdf
         @uuid = SecureRandom.uuid
         @request_body = {
           'document' => to_faraday_upload,
           'metadata' => generate_metadata
         }
+        @submission = submission_id ? Form526Submission.find(submission_id) : nil
```

Comment:
Simplify using `Form526Submission.find_by_id(submission_id)`, instead of the conditional with `find`, to return the same nil result if the record is not found.